go.mod

```diff
@@ -1,5 +1,5 @@
 module github.com/mattkimber/gorender
 
-go 1.15
+go 1.21
```

Comment:
Needed for `min` and `max`. Can also be easily circumvented by implementing these ourselves, but 1.21 wasn't exactly a new version either.